internal/injector/builtin/yaml/stdlib/runtime.yml

```
@@ -46,3 +46,30 @@ aspects:
     advice:
       - prepend-statements:
           template: getg().__dd_gls = nil
+
+  # Temporarily hack tracer.SpanFromContext so that it no longer returns true if
+  # the Span found from GLS is actully a noop span.
```

```suggestion
  # Temporarily hack tracer.SpanFromContext so that it no longer returns true if
  # the Span found from GLS is actually a noop span.
  # This might happen when a library creates spans using the OTel SDK.  
```

[nit] just adding a little bit more context for the future